src/tests/FunctionalTests/iOS/Simulator/LambdaCompilerAOT/iOS.Simulator.LambdaCompilerAot.Test.csproj

```diff
@@ -11,6 +11,7 @@
     <IncludesTestRunner>false</IncludesTestRunner>
     <ExpectedExitCode>42</ExpectedExitCode>
     <EnableAggressiveTrimming>true</EnableAggressiveTrimming>
+    <DynamicCodeSupport>false</DynamicCodeSupport>
```

Comment:
I assume this setting needs to be there because we are testing the full AOT mode?
It would be good to move it to some common .props/targets file instead as adding new tests for testing full AOT could miss setting this feature switch.

Reply:
Correct. Updated condition in `tests.mobile.targets` to override default trimming features.